packages/eas-cli/src/commands/channel/create.ts

```diff
@@ -122,22 +114,22 @@ export default class ChannelCreate extends Command {
     try {
       const existingBranch = await getBranchByNameAsync({
         appId: projectId,
-        branchName: channelName,
```

Comment:
branchName and channelName are different things right? shouldn't this be the name of a branch instead of a channel?